src/gino/dialects/asyncpg.py

```diff
@@ -453,16 +492,22 @@ class AsyncpgDialect(PGDialect, base.AsyncDialectMixin):
 
     def __init__(self, *args, **kwargs):
         self._pool_kwargs = {}
+        self._init_hook = None
         for k in self.init_kwargs:
             if k in kwargs:
-                self._pool_kwargs[k] = kwargs.pop(k)
+                if k == "init":
```

Comment:
What's the use case of `init`? I don't see it documented or used except tests

Reply:
Oh, this is an asyncpg parameter on [`create_pool()`](https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.pool.create_pool). I'd also expect we could refactor this to be more generic (using SQLAlchemy [`pool_size`](https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.QueuePool.params.pool_size) and [`max_overflow`](https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.QueuePool.params.max_overflow) for example) in SQLAlchemy 1.4/2.0 support.